gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/controller/GraphControllerTest.java

```diff
@@ -89,14 +90,16 @@ void getStoretypes_ReturnsStoretypesAsList_whenSuccessful() throws Exception {
 
     @Test
     void getGraphs_ReturnsGraphsAsList_whenSuccessful() throws Exception {
+        LocalDateTime currentTime = LocalDateTime.now();
+        LocalDateTime tenDaysLater = currentTime.plusDays(10);
         final GaaSGraph graph = new GaaSGraph()
                 .graphId(TEST_GRAPH_ID)
                 .description(TEST_GRAPH_DESCRIPTION)
                 .url("my-graph-namespace.apps.k8s.cluster/ui")
                 .restUrl("my-graph-namespace.apps.k8s.cluster/rest")
                 .configName("mapStore")
                 .status(RestApiStatus.UP)
-                .graphAutoDestroyDate("2022-06-09t15:55:34.006");
+                .graphAutoDestroyDate(tenDaysLater.toString());
```

Comment:
This used to be in the past but now in the future?

Reply:
graphAutoDestroyDate field is  when graph should be deleted. Yes it should be always future.